src/bridge/MatrixHandler.ts

```diff
@@ -1066,14 +1069,43 @@ export class MatrixHandler {
                 cacheBody = reply.reply;
             }
         }
+
+        // special handling for edits
+        if (event.content["m.relates_to"] && event.content["m.relates_to"].rel_type === "m.replace") {
```

Comment:
I forget if TS likes these?

```suggestion
        if (event.content.["m.relates_to"]?.rel_type === "m.replace") {
```

Reply:
Seems like it does – doesn'teven need a `.` Fixed in b7df6011